hocr-extract-images

```diff
@@ -13,7 +13,7 @@ def assoc(key,list):
 
 def get_text(node):
     textnodes = node.xpath('.//text()')
-    s = string.join([text for text in textnodes])
+    s = ''.join([text for text in textnodes])
```

Comment:
I guess we can have it also in this commit here. The only reason that I see is that clean commits for exactly one thing are best if at some point we may have to revert some of them. Let us be optimistic, we don't have to do that (and even if, then Travis will show us an error here).